torch/fx/experimental/proxy_tensor.py

```diff
@@ -393,20 +479,33 @@ def make_fx(f, decomposition_table=None, trace_factory_functions=True, use_fake=
     def wrapped(*args):
         phs = pytree.tree_map(lambda _: fx.PH, args)  # type: ignore[attr-defined]
         fx_tracer = PythonKeyTracer()
-        fake_tensor_mode = FakeTensorMode() if use_fake else nullcontext()
+        fake_tensor_mode = FakeTensorMode(allow_fallback_kernels=False) if use_fake else nullcontext()
```

Comment:
It would be nice if we could still use fallback kernels if there are no symbolic values